plugins/experimental/sslheaders/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 #######################
 
 add_library(sslhdr STATIC expand.cc util.cc)
-target_link_libraries(sslhdr PRIVATE OpenSSL::SSL ts::tsutil)
+target_link_libraries(sslhdr PRIVATE OpenSSL::SSL libswoc::libswoc)
```

Comment:
It seems swoc isn't included anywhere. Why is it needed here?

Reply:
something includes libswoc headers

Reply:
There may be an incorrect dependency somewhere else.